modin/utils.py

```diff
@@ -296,19 +337,25 @@ def _replace_doc(
 
     if parent_cls and isinstance(target_obj, property):
         if overwrite:
-            target_obj.fget.__doc_inherited__ = True
+            target_obj.fget.__doc_inherited__ = True  # type: ignore
+        assert attr_name is not None
         setattr(
             parent_cls,
             attr_name,
             property(target_obj.fget, target_obj.fset, target_obj.fdel, doc),
         )
     else:
         if overwrite:
-            target_obj.__doc_inherited__ = True
+            target_obj.__doc_inherited__ = True  # type: ignore
```

Comment:
Why ignore?

Reply:
Without this, we run into `modin/utils.py:349:13: error: "object" has no attribute "__doc_inherited__"  [attr-defined]`